docs/management/rollups/create_and_manage_rollups.asciidoc

```diff
@@ -165,7 +165,7 @@ as your source to see both the raw and rolled up data.
 [role="screenshot"]
```

Comment:
@dej611 INLINE EDITOR TESTING

Here's some additional issues I found while testing this feature within the inline editing context:

* Chart picker size does not stretch to full width when no stacking is available in inline panel
  
  <img width="951" alt="Screenshot 2024-08-27 at 15 52 20" src="https://github.com/user-attachments/assets/d4212148-3aea-4108-8e5a-e6f680c83882">
  <img width="958" alt="Screenshot 2024-08-27 at 15 51 40" src="https://github.com/user-attachments/assets/91af9e3f-113c-4699-ab56-b7a45ed25758">
  <img width="1507" alt="Screenshot 2024-08-27 at 15 50 27" src="https://github.com/user-attachments/assets/d45acdf5-fbf4-4019-95ce-8a236fc8199a">
  <img width="399" alt="Screenshot 2024-08-27 at 15 50 16" src="https://github.com/user-attachments/assets/37d10ec7-b28c-4009-bd7d-2df3769840a1">
  <img width="1148" alt="Screenshot 2024-08-27 at 15 50 04" src="https://github.com/user-attachments/assets/406ecf08-2a2b-4b53-93c8-e9ac1b535d5a">

* When stacked control is enabled the problem is the big gap between the two controls
  <img width="655" alt="Screenshot 2024-08-27 at 15 51 20" src="https://github.com/user-attachments/assets/c74da935-a4de-4c85-a80b-7604e5463c79">

Reply:
Note: I reproduce all of the above within the `dev examples` environment. I've tried to reproduce them again in regular mode and could not reproduce. This lowers the importance of the issues.

Reply:
Cannot reproduce it even with dev examples so I'm resolving it for now. If it comes back, I'll keep an eye on it.